server/writango/admin.py

```diff
@@ -3,7 +3,7 @@
 
 from django.contrib import admin
 from django.contrib.auth.models import User
-from models import *
+from writango.models import *
```

Comment:
Explicit & cleaner. Thanks
For some reasons, I couldn't reproduce the `no module found` error though.